config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform_withCache.yaml

```diff
@@ -6,10 +6,11 @@ spec:
   cluster: kubernetes
   platform:
     publishStrategy: "Kaniko"
-    baseImage: quay.io/kiegroup/kogito-swf-builder-nightly:latest
+    baseImage: quay.io/kiegroup/kogito-swf-builder-nightly
```

Comment:
Please, this is for use only on prod. Devmode should have its own field. We've already clarified this.

Reply:
ok